src/client/decorators/MetaDecorator/DiagramDesigner/templates/AttributeDetailsDialog.html

```diff
@@ -34,6 +41,12 @@ <h3>Edit attribute parameters...</h3>
                             Read-only
                         </label>
                     </div>
+                    <div id="pHidden" class="col-sm-2 controls" title="If true, value will not be visible in the property editor">
```

Comment:
`If true, value will not be visible in the property editor` -> `If true, the attribute will not be displayed in the GUI`